data_expansion2/localized/de_patch/text/conversations/px2_05_wildernesses/px2_05_si_frontal_assault.stringtable

```diff
@@ -322,7 +322,7 @@ Die überraschende Geschwindigkeit hilft euch. Ihr rennt mit dem Rammbock auf da
     </Entry>
     <Entry>
       <ID>68</ID>
-      <DefaultText>Du hast einen Atemzug Zeit, um zu entscheiden, welchem deiner Gefährten du zur Seite springst&#160;…</DefaultText>
```

Comment:
Naja ^^´´ Springen und schubsen sind nun wirklich zweierlei aktionen. In der Szene greift die Gruppe gerade eine Festung an und es kommt heißes Pech von oben geschwabt. Aber ok. Kann ich damit leben.